mowgli_etl/pipeline/wdc/data_exploration/offers_corpus_english_v2_random_sample.py

```diff
@@ -0,0 +1,20 @@
+from random import sample
+from pathlib import Path
+
+from mowgli_etl.pipeline.wdc.wdc_offers_corpus_entry import WdcOffersCorpusEntry
+
+def data_sample(wdc_jsonl_file_path: Path, n: int) -> WdcOffersCorpusEntry:
```

Comment:
Rather than have a freestanding function, create a WdcOffersCorpus class with (jsonl_file_path: Path) constructor and have a sample(n: int) method on it.